src/NLogTarget/ApplicationInsightsTarget.cs

```diff
@@ -49,6 +50,37 @@ internal TelemetryClient TelemetryClient
             get { return this.telemetryClient; }
         }
 
+        internal void BuildPropertyBag(LogEventInfo logEvent, ITelemetry trace)
+        {
+            trace.Timestamp = logEvent.TimeStamp;
+            trace.Sequence = logEvent.SequenceID.ToString(CultureInfo.InvariantCulture);
+
+            IDictionary<string, string> propertyBag;
+
+            if (trace is ExceptionTelemetry)
+            {
+                propertyBag = ((ExceptionTelemetry)trace).Properties;
+            }
+            else
+            {
+                propertyBag = ((TraceTelemetry)trace).Properties;
+            }
+
+            if (!string.IsNullOrEmpty(logEvent.LoggerName))
+            {
+                propertyBag.Add("LoggerName", logEvent.LoggerName);
```

Comment:
Consider `nameof(logEvent.LoggerName)`

Reply:
I'm just moving code around to fix the unit tests.

Reply:
Fine to leave it as it is, then.